svc-bip-api/src/main/java/gov/va/vro/bip/config/RabbitMqConfig.java

```diff
@@ -73,4 +73,24 @@ public void configureRabbitListeners(RabbitListenerEndpointRegistrar registrar)
   DirectExchange bipApiExchange() {
     return new DirectExchange(exchangeName, true, true);
   }
+
+  @Bean
+  FanoutExchange deadLetterExchange() {
+    log.info("Creating dead letter exchange with name={}", props.getDeadLetterExchangeName());
```

Comment:
The log statements in these beans duplicate log statements already output by the `RabbitAdmin` on application startup. The look like this:
```
2024-08-07T15:33:22.540-07:00  INFO 67182 --- [           main] o.s.amqp.rabbit.core.RabbitAdmin         : Auto-declaring a non-durable or auto-delete Exchange (bipApi.dlx) durable:true, auto-delete:true. It will be deleted by the broker if it shuts down, and can be redeclared by closing and reopening the connection.
```

Reply:
My team wanted these added as a failsafe in case the configuration is botched and RabbitMQ isn't logging those details